### PR TITLE
internationalization for Chrome Ext

### PR DESCRIPTION
**Pull request by ArtemNikolaev**
#1

### CODE REVIEW

File: src/services/passwordGenerationService.js [hunk 1]
```diff
@@ -99,39 +99,8 @@ function initPasswordGenerationService() {
...
-        // Use & to apply the mask and reduce the number of recursive lookups
-        rval = rval & mask;
-
-        if (rval >= range) {
-            // Integer out of acceptable range
-            return randomInt(min, max);
-        }
-        // Return an integer that falls within the range
-        return min + rval;
+        return Math.floor(Math.random() * (max - min + 1)) + min;
```
Comment (kspearrin) [inline src/services/passwordGenerationService.js]: Looks like you had a bad merge or something. This needs to be undone.

File: src/popup/index.html [hunk 2]
```diff
@@ -41,6 +41,7 @@
 
     <script src="app/services/servicesModule.js"></script>
     <script src="app/services/backgroundService.js"></script>
+    <script src="app/services/i18n.js"></script>
```
Comment (kspearrin) [inline src/popup/index.html]: I don't see the i18n service file anywhere?

Comment (ArtemNikolaev) [inline src/popup/index.html]: I forget to add new files in the git, sorry ((. Now everything will be fine.

File: src/_locales/en/messages.json [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+  },
+  "masterPass": {
+    "message": "Master Password",
+    "description": "Master Password"
+  },
+  "masterPassDesc1": {
+    "message": "The master password is the password you use to access your vault. It is very important that you do not.",
+    "description": "The master password is the password you use to access your vault. It is very important that you do not."
+  },
+  "masterPassDesc2": {
```
Comment (kspearrin) [inline src/_locales/en/messages.json]: This one seems to be broken into two sentences incorrectly.

> It is very important that you do not forget your master password.

Is one sentence. This breaks it into two.

`masterPassDesc1` and `masterPassDesc2` could just  all be one message.

Comment (ArtemNikolaev) [inline src/_locales/en/messages.json]: fix it

File: src/popup/app/tools/views/toolsPasswordGenerator.html [hunk 4]
```diff
@@ -3,9 +3,9 @@
         <a ng-click="close()" href="">Close</a>
     </div>
     <div class="right">
-        <a ng-click="select()" ng-show="showSelect" href="">Select</a>
+        <a ng-click="select()" ng-show="showSelect" href="">{{ i18n.select</a>
```
Comment (kspearrin) [inline src/popup/app/tools/views/toolsPasswordGenerator.html]: Looks like missing `}}` here

Comment (ArtemNikolaev) [inline src/popup/app/tools/views/toolsPasswordGenerator.html]: Yes, sorry my carelessness (( I will fix it right now

**Pull request merged**